Modules/Numerics/FEM/include/itkFEMLinearSystemWrapper.h

```diff
@@ -583,7 +583,7 @@ class ITK_ABI_EXPORT FEMExceptionLinearSystemBounds : public FEMException
   ~FEMExceptionLinearSystemBounds() noexcept override;
 
   /** Type related information. */
-  itkOverrideGetNameOfClassMacro(FEMExceptionLinearSystem);
+  itkOverrideGetNameOfClassMacro(FEMExceptionLinearSystemBounds);
```

Comment:
This has already been fixed on `master`: see #4398. Please, rebase on `master`.

Reply:
done